pkg/contexts/credentials/repositories/npm/config_test.go

```diff
@@ -26,8 +26,8 @@ var _ = Describe("Config deserialization Test Environment", func() {
 		spec := npm.NewRepositorySpec("testdata/.npmrc")
 
 		_ = Must(ctx.RepositoryForSpec(spec))
-		id := identity.GetConsumerId("registry.npmjs.org", "pkg")
-
+		id, err := identity.GetConsumerId("registry.npmjs.org", "pkg")
+		Expect(err).To(BeNil())
```

Comment:
you could use `Must` here, similar to the line below

Reply:
done